Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts

```diff
@@ -209,7 +207,7 @@ export class AzureAppService {
         var isNewValueUpdated: boolean = false;
         for(var key in addProperties) {
             if(applicationSettings.properties[key] != addProperties[key]) {
-                tl.debug(`old value : ${applicationSettings.properties[key]}. new value: ${addProperties[key]}`);
+                tl.debug(`Value of ${key} has been changed`);
```

Comment:
Print updated value. Might be relevant info for customer.

Reply:
`Value of ${key} has been changed to ${addProperties[key]}`

Reply:
It is done.